ambry-cloud/src/test/java/com.github.ambry.cloud/CloudDataNodeTest.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2019 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.cloud;
+
+import com.github.ambry.config.CloudConfig;
+import com.github.ambry.config.ClusterMapConfig;
+import com.github.ambry.config.VerifiableProperties;
+import com.github.ambry.network.PortType;
+import java.util.Properties;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+
+/**
+ * Test class testing behavior of {@link CloudDataNode}.
+ */
+public class CloudDataNodeTest {
+
+  private final ClusterMapConfig clusterMapConfig;
+  private final CloudConfig cloudConfig;
+
+  public CloudDataNodeTest() {
+    Properties props = new Properties();
+    props.setProperty("clustermap.host.name", "localhost");
+    props.setProperty("clustermap.cluster.name", "clusterName");
+    props.setProperty("clustermap.datacenter.name", "DC1");
+    props.setProperty("clustermap.ssl.enabled.datacenters", "DC0,DC1");
+    props.setProperty("clustermap.port", "12300");
+    clusterMapConfig = new ClusterMapConfig(new VerifiableProperties(props));
+
+    props = new Properties();
+    props.setProperty("vcr.ssl.port", "12301");
```

Comment:
Use static strings.

Reply:
I checked other config files, looks like no static string defined for config name. They are only spelled in test, which I think should be ok.